File: 1737-maximum-nesting-depth-of-the-parentheses/1737-maximum-nesting-depth-of-the-parentheses.py
class Solution:
    def maxDepth(self, s: str) -> int:
        max_cnt = -1
        cnt = 0
        for c in s:
            if c=="(":
                cnt+=1
            elif c==")":
                cnt-=1
            max_cnt = max(max_cnt, cnt)
        return max_cnt

File: 1737-maximum-nesting-depth-of-the-parentheses/1737-maximum-nesting-depth-of-the-parentheses.java
class Solution {
    public int maxDepth(String s) {
        int maxCount = -1;
        int cnt = 0;
        s.toCharArray();
        for (char c: s.toCharArray()){
            if(c == '('){
                cnt++;
            }else if(c == ')'){
                cnt--;
            }
            maxCount = Math.max(maxCount, cnt);;
        }
        return maxCount;
    }
}